app/Models/Sponsor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Models;
+
+use App\Eloquent\Model;
+use Illuminate\Database\Schema\Blueprint;
+use Orbit\Concerns\Orbital;
+use Spatie\EloquentSortable\Sortable;
+use Spatie\EloquentSortable\SortableTrait;
+use Spatie\Sluggable\HasSlug;
+use Spatie\Sluggable\SlugOptions;
+
+class Sponsor extends Model implements Sortable
+{
+    use SortableTrait;
```

Comment:
That trait and all it's related stuff can also be removed.

Reply:
Fixed